test/functional/core/job_spec.lua

```diff
@@ -511,7 +565,12 @@ describe('jobs', function()
 
     it('will return status codes in the order of passed ids', function()
       source([[
-      call rpcnotify(g:channel, 'wait', jobwait([
+      call rpcnotify(g:channel, 'wait', jobwait(has('win32') ? [
+      \  jobstart('Start-Sleep -Milliseconds 70; exit 4'),
+      \  jobstart('Start-Sleep -Milliseconds 50; exit 5'),
+      \  jobstart('Start-Sleep -Milliseconds 30; exit 6'),
+      \  jobstart('Start-Sleep -Milliseconds 10; exit 7')
```

Comment:
This is going to be unreliable because powershell startup takes 600ms at least.

Reply:
Bump the sleep timeout to 700, 500, 300, 100 or use cmd.exe and ping instead?
The timeout test for jobwait lasts up to 2-3s so it's long wait either way.

Reply:
I think cmd.exe would be a better choice here.

Reply:
Likely because of Powershell's lazy-loading and Appveyor-specific modules. Can't prevent them with `-NoProfile`.